packages/salesforcedx-vscode-core/src/notifications/notificationService.ts

```diff
@@ -84,13 +84,12 @@ export class NotificationService {
     cancellationToken?: vscode.CancellationToken
   ) {
     observable.subscribe(async data => {
-      if (data !== undefined && data.toString() === '0') {
+      if (data !== undefined && data !== null && data.toString() === '0') {
```

Comment:
To simply, you can just use `String(data)`

Reference: https://exploringjs.com/impatient-js/ch_strings.html#converting-to-string

Reply:
sounds good. Thanks!!!